### PR TITLE
firx multiple issues

### LINKED ISSUE #114: Filtering added Clarifai's data on the dropdown/popUp in accordance to available Data (aka Alastair's)
What needs to be done:

- [x] Add clarifai's data to the drop-down. 

- [x] Make JSON file of Alastair's data.

- [x] Call `ItemType` from Airtable on `GetDetails` component's render. 

- [x] Filter clarifai's data on the frontend to prevent duplicates.

### LINKED ISSUE #171: Add star for compulsory fields in Get Details

### LINKED ISSUE #172: Change button text in get details to login to save data

### LINKED ISSUE #173: add ‘multi-coloured’ option to colors select tag in Form component FE

### PR DESCRIPTION
**Pull request by amanshawar**
filtered color data fix #114 
add colored option  fix #173
change button name fix #172 
add star in get details fix #171

fix colors in details of item

### CODE REVIEW

**Review by FidaaHersh — APPROVED** [review]

File: client/src/components/Shared/Form/index.js [hunk 1]
```diff
@@ -16,196 +16,216 @@ import {
...
-      selected_details,
-      itemType,
-      colors,
-      brands,
-      condition,
-      labelSize,
-      age,
-      showDefaultOption,
-    } = this.props;
+circle = hex => (
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js]: Please delete this!

File: client/src/components/Shared/Form/form.style.js [hunk 2]
```diff
@@ -22,6 +22,13 @@ export const StyledImg = styled.img`
   height: 200px;
 `;
 
+export const CircleColor = styled.div`
```
Comment (jema28) [inline client/src/components/Shared/Form/form.style.js]: Please delete this.

File: client/src/components/Shared/Form/index.js [hunk 3]
```diff
@@ -16,196 +16,216 @@ import {
   StyledInput,
   StyledTextarea,
   StyledImgCon,
+  CircleColor,
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js]: Please delete this.

**Review by jema28 — CHANGES_REQUESTED** [review]
Half reviewed - please delete all references to the circle 🔴

**Review by Hadeel-Salamin — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

File: client/src/data.json [hunk 4]
```diff
@@ -4,7 +4,21 @@
     { "id": "recMr78NZa2akmLW0", "name": "ASOS" },
     { "id": "recqts81sMcIoFMMJ", "name": "Adidas" }
   ],
-  "colors": ["white", "black", "red", "blue", "green", "limegreen", "gray"],
+  "colors": [
```
Comment (jema28) [inline client/src/data.json:7]: Replace these colours with his list.

**Review by jema28 — CHANGES_REQUESTED** [review]

**Review by jema28 — APPROVED** [review]

**Review by Hadeel-Salamin — APPROVED** [review]

**Pull request merged**